WeakAuras/Prototypes.lua

```diff
@@ -6711,7 +6711,15 @@ Private.event_prototypes = {
       trigger.itemName = trigger.itemName or 0;
       local itemName = type(trigger.itemName) == "number" and trigger.itemName or "[["..trigger.itemName.."]]";
       local ret = [[
-        local count = GetItemCount(%s, %s, %s);
+        local itemName = %s
+        local count = GetItemCount(itemName, %s, %s);
```

Comment:
I don't think that works if people have the same item in multiple qualities?

Reply:
It seems i made wrong assumption that all items with different quality have different itemId
It is the case with Refreshing Healing Potion (R1 & R3 does not have same id), Pigments, and flowers
But not with profession equipments and probably more items